src/GitVersionCore.Tests/VariableProviderTests.cs

```diff
@@ -1,15 +1,14 @@
-using System;
+using GitVersion;
 using GitVersion.Logging;
-using NUnit.Framework;
-using Shouldly;
 using GitVersion.OutputFormatters;
 using GitVersion.OutputVariables;
-using GitVersion;
 using GitVersion.VersionCalculation;
 using GitVersion.VersioningModes;
+using NUnit.Framework;
+using Shouldly;
+using System;
 
-namespace GitVersionCore.Tests
-{
+namespace GitVersionCore.Tests {
```

Comment:
Please don't deviate from code formatting

Reply:
Sorry its how my visual studio has been configured. Does it auto on save.
I fixed it with the github editor.